src/components/where-to-buy/WhereToBuyContent.tsx

```diff
@@ -0,0 +1,49 @@
+"use client"
+
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs"
+import PhysicalStores from "@/components/where-to-buy/PhysicalStores/PhysicalStores"
+import OnlineStores from "@/components/where-to-buy/OnlineStores/OnlineStores"
+import { useRouter, useSearchParams } from "next/navigation"
+import { useTranslations } from "next-intl"
+
+type WhereToBuyContentProps = {}
+
+const tabs = ["physical-store", "online-platform"]
+
+export default function WhereToBuyContent({}: WhereToBuyContentProps) {
+  const router = useRouter()
+  const searchParams = useSearchParams()
+  const tWhereToBuy = useTranslations("where-to-buy-landing")
+  const defaultTab = searchParams.get("type") || "physical-store"
+
+  const handleTabChange = () => {
+    if (defaultTab === tabs[0]) {
+      router.replace(`?type=${tabs[1]}`, { scroll: false })
+    } else router.replace(`?type=${tabs[0]}`, { scroll: false })
+  }
```

Comment:
change tabs to something like this:
const tabs = {
  physicalStore: "physical-store",
  onlinePlatform: "online-platform",
};

for better readability,

or use link to wrap TabTrigger and implement the logic there